### PR TITLE
Ana/fix rewards crawler

### LINKED ISSUE #61: Check rewards crawler not saving in database in new Backend
Check why rewards crawler is failing to store data to Hasura database

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #61 (should close)

**Description:**

Researching what Mario said I discovered the previous queries that were working are not working anymore (currentcurrentEraPointsEarned and currentElected).

Here I am attempting to fix them.

EDIT: for now, I updated the @polkadot/api package and used what I think are the queries from the [source code](https://github.com/polkadot-js/api/blob/5b40d95b0f9c57cab8a326b79cd241533552c13f/packages/api-derive/src/staking/overview.ts), although I get some weird objects as responses.

Thank you! 🚀

---

For contributor:

- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by mariopino** [comment]
Related: https://github.com/polkadot-js/api/issues/2040

**Comment by mariopino** [comment]
Based on the info provided by @jacogr we need to rewrite this! :-)

File: lib/crawlers/rewards.js [hunk 1]
```diff
@@ -66,9 +66,14 @@ module.exports = {
           let value = event.attributes.attributes[0].value;
           let end_era_block_id = event.attributes.block_id - 1;
           let hash = await api.rpc.chain.getBlockHash(end_era_block_id);
-          let eraPoints = await api.query.staking.currentEraPointsEarned.at(hash.toString());
-          let endEraValidatorList = await api.query.staking.currentElected.at(hash.toString());
-          const session_index = await api.query.session.currentIndex.at(hash);
+          // let eraPoints = await api.query.staking.currentEraPointsEarned.at(hash.toString());
+          // let endEraValidatorList = await api.query.staking.currentElected.at(hash.toString());
+          // const session_index = await api.query.session.currentIndex.at(hash);
+          let currentEraIndex = await api.query.staking.currentEra() // O.o no clue. This returns this: Option { registry: TypeRegistry {}, raw: <BN: 213> }
```
Comment (mariopino) [inline lib/crawlers/rewards.js]:
```suggestion
         let activeEra = await api.query.staking.activeEra();
         activeEra = JSON.parse(JSON.stringify(activeEra));
         const currentEraIndex = activeEra.index;
```

Comment (Bitcoinera) [reply lib/crawlers/rewards.js]: > We need a lot of changes here!

Okidoki

**Comment by Bitcoinera** [comment]
> exposures.forEach(([key, exposure]) => {
>   console.log('key arguments:', key.args.map((k) => k.toHuman()));
>   console.log('     exposure:', exposure.toHuman());
> });
> ```

I get `toHuman is not a function` :woman_shrugging:

**Comment by Bitcoinera** [comment]
This is now working!!! :muscle: 

At least with local Postgres. Will try with the dockers

**Comment by mariopino** [comment]
Hey @Bitcoinera!

I tested your branch and great! ^^ We are getting some rows in the rewards table!

**Some observations**

Seems that the data we are saving is not the one we need as the goal of this crawler is to get **slashes** and **rewards** per **validator/era** but:

- **commission** is not saved
- **era_rewards** should store the rewards for the validator/era in KSM (currently is saving `[object Object] ...`)
- **stake_info** should save the staking info for the validator/era (currently is saving era points for all validators ???)
- **stash_id** is saving the address with quotes.
- **era_slashes** is not saved (also it's not in table schema, we need to add it). This is one of the two main goals of this crawler.

```
{
  "data": {
    "rewards": [
      {
        "commission": null,
        "era_rewards": "\"[object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object][object Object]\"",
        "stake_info": "{\"total\":53000,\"individual\":{\"CdEm1ErGKML3waXabLvn3NyqdAGXBQJVngLaM86YM5Yb9dr\":280,\"Cdaq3iUobZLYD2d8oqRQf1VKuRo6H64KsMBgJyxap4ZnARX\":220,\"CinNnPhc4aGFQb7FWhUpnfwCNwXN3brcnCR1MkazkstcDJa\":180,\"CmD9vaMYoiKe7HiFnfkftwvhKbxN9bhyjcDrfFRGbifJEG8\":200,\"CmfwTdJ1F7Qd2HLAQ6A8FdNtz9hKvTMr2FkWtBQR1vECYjo\":440,\"CoooBnE4hYaaQSKUDyCmZNdn8uE94rWa5ez3jrpFDE88vVA\":240,\"CoqysGbay3t3Q7hXgEmGJJquhYYpo8PqLwvW1WsUwR7KvXm\":380,\"CrzGYAYYnguxoR5pGx4UbwLs2DkoxoLiLJd8kjZMQzDuq8r\":320,\"Cs1jHXYHxZyWKsGPvYY1BknhLdpNa1iXGja4Kn8hQZs7BsH\":240,\"Cum9b4v9pAhFXQXy3c44ehJ9gi6Bq2rPdZcyT85vzBa8qWc\":300,\"Cwr82cUKvfgnQSnLeN84CmtNaUGjYBD2qaBjsTxG43ZNh2M\":300,\"D3JqthKZjAEge84RQ3dheuQmaX2nVhk9A3sHY1rGNbStiFg\":240,\"D5NGokAYvFZmvFynjApN2DG4r1H1cQ9pyum7KC3KmtDYLkn\":280,\"D8BfryaM5xN62UuKUpLK5zbZEUSBtA76yP9YddQTKXi9pkB\":240,\"D948vxMSA6u7G5gqPGQdAUDMJbiR7wgqZ1La8XeBiXr9FTF\":320,\"DAT4gSgyMskggCmTQKfEM6hQgRy1NWdhtfMTAuPLsUAGUPy\":260,\"DDkGq46ftxusKM4n4jfEr7MBW1DPGuVQDeExMFQbRtnsenD\":420,\"DEFQigjrErCANqTMiTRuy7BB5L46q9C36Bxqe41GV1yR6vA\":340,\"DFaiE6wT1caQ9u7eLkuphVQqbywWKzYjm5vRsMgk43GSb84\":360,\"DGC4rANssfdpry7FQFznHo2b3QbDHyQZM58esJU6RskB41y\":440,\"DGiLC1uvohfgHiirDtsr5HJRYXUFy9NdXj6YC6azcaRpvJZ\":200,\"DKUQiUWNPGvGYrgaxWqduJVRqWUYXUeKprX9EJENhxYvVyS\":300,\"DKinge6g7FNNj3hJZJUze6GByxkLR7ipCqrnuSQKGRMg37G\":240,\"DSpbbk6HKKyS78c4KDLSxCetqbwnsemv2iocVXwNe2FAvWC\":220,\"DTLcUu92NoQw4gg6VmNgXeYQiNywDhfYMQBPYg2Y1W6AkJF\":200,\"DTMCVLVqNj5L3kpuftg49bY9xozwhRyF52sEHdBySbpbPkt\":340,\"DaCSCEQBRmMaBLRQQ5y7swdtfRzjcsewVgCCmngeigwLiax\":260,\"DbAdiLJQDFzLyaLsoFCzrpBLuaBXXqQKdpewUSxqiWJadmp\":360,\"DbuPiksDXhFFEWgjsEghUypTJjQKyULiNESYji3Gaose2NV\":260,\"DfiSM1qqP11ECaekbA64L2ENcsWEpGk8df8wf1LAfV2sBd4\":400,\"Dg5QS2TKCMQbbZau7acRXHPN4xQxqqase8awmzLXXgmXb7w\":300,\"DiCVHECatzBxCKZeCmQhvqbAo9KK6CHubAWtQF85N3YbKFW\":360,\"DimyMqRfrnqudRLVf5TxAM3T7X23PKvFDp85DDGabFKMQ2a\":400,\"DokiayXWoMvotzchNdLSH4P4Fe7EvMESZvZL4Fn3NekoFtf\":260,\"DrQHiQu5VkaRuv1H3iELXVqsvD3SV3E8xNjJqXUgECSg23R\":260,\"DriCrAgdVV57NeQm5bWn5KQpVndVnXnm55BjRpe6qzZ5ktJ\":300,\"DrkY92Yq67zJ7T8XWN7SAXbnNDJhzkADutwhhhPyW3tQKTw\":320,\"DwZBcfHnJtRmR7P23VgHQEzaeGPXQvH8jDvuob2qiyTHJMM\":280,\"E2ZKmzMzajqW838jXVSM5DyoUJUdEQddXNknEjoTwj2zBLj\":460,\"E457XaKbj2yTB2URy8N4UuzmyuFRkcdxYs67UvSgVr7HyFb\":260,\"E58yuhUAwWzhn2V4thF3VciAJU75eePPipMhxWZe9JKVVfq\":140,\"E8QEnkMMyWWrHPTbA7jo547SpkoGcE6yoTXHmrQeXY2sGnT\":160,\"EFjHdypk8xLf3ocDEFPaKFWVcfamH8mpvfUeXHvRWpSBk2M\":220,\"EGZEgucx8vTCSak5uBwwkPkZ7FiYCWYmhpMjWFupdMAq2BN\":400,\"ESaTdKYYtRYhJbYjMzgKauNyZHbaNzx2vknC9eioqKusWSt\":260,\"EUoo6xidm7okewmbh89tUW3sqaDLYKAvTCCNYc8zh4nPU4s\":220,\"EXkCSUQ6Z1hKvGWMNkUDKrTMVHRduQHWc8G6vgo4NccUmhU\":320,\"EXtQYFeY2ivDsfazZvGC9aG87DxnhWH2f9kjUUq2pXTZKF5\":460,\"EY35xeDhXibMDS5GmdHM2UzpvT4VoV9fGXDy3Muq6cyLPja\":320,\"EZ7uBY7ZLohavWAugjTSUVVSABLfad77S6RQf4pDe3cV9q4\":160,\"EafgFRX24PTgJAjGoaDuQLXQiLX4daSFQQttzGVticSD18o\":300,\"Ed6JFR7JmeGtmrubAbXtJRjq9FPWhGRWMdmYnLRcsqaabT8\":320,\"EdEQVcrr4pvzDLdb5xFkncKYhCE6uyuE1eKRTqg4PDvBgMa\":300,\"EfK27sX89DpagD3TCF4hF4rGZ1CnCGtYZvo94HZLU3GQuMj\":380,\"EibDgpnEGwqvWDcPUq7EThEB6kPEnqKuZog48E5jga8uWe8\":360,\"EicrAEbyauqktQpp4CdvsF2CQy3Ju7tGGMohj3h5sAPnKHL\":400,\"Eo4boG437k7gFy75VqPrWP5gHSGi9Sm6CZzLFFsCGYaSPzM\":300,\"Eo9RxTKq2WppUvRRycUmLFHJvHtBVoxURhDe78ZKmjoJfWN\":280,\"Eodfj4xjkw8ZFLLSS5RfP6vCMw8aM6qfM7BfeQMf6ivFWHy\":300,\"EoeAhrhJv2p6Sg5T67mxkLwrPnjKbH7iZSh8q1JTyvELvrR\":340,\"EonK7NScfhd7ZRfgnLhm4cKRFJWK1z59zPximUZRg8VjHQj\":320,\"EqyCQvYn1cHBdzFVQHQeL1nHDcxHhjWR8V48KbDyHyuyCGV\":320,\"ErVvRnhXrJS7W9rTtnDECpZTFyLACtDcXEPtzEyYw4b8zcg\":320,\"Et5ne6GRrWZ178npoYdnG8RRU92gzqBiSRf4hTaU3Yonsf2\":340,\"Etij9aH36W1NjjWbR7wB5j41CmfpqAx8D4V4HCJhUydSH9Y\":340,\"EviBmmkq5o5ak2jUo71SPuDEdwGTr8Yz8U5tXBgF6fy8kKV\":180,\"EyibGsAttxpNBkgjMxNTArskxkdEFFbwghYuuaZyvu9rmo2\":380,\"Eyvj7oeaHyoqJbNPKPi3zjBks2bNoju3X29xJhsxD8d6GeQ\":340,\"EzR9J3Afvash2tYCk8ZZwPYyq3zy92adVUXKcYjbYN46JWL\":320,\"F2GF2vuTCCmx8PnUNpWHTd5hTzjdbufa2uxdq2uT7PC5s9k\":200,\"F4LocUbsPrcC8xVap4wiTgDakzn3xFyXneuYDHRaHxnb6dH\":420,\"F4xrhkWsW2PSqZBuBMVJGE3LQy7R2ZWo3J8d6KEsDznrDF8\":260,\"F7BeW4g5ViG8xGJQAzguGPxiX9QNdoPNc3YqF1bV8d9XkVV\":300,\"F7Nnn8nQL3Bjxap8Ax1HDZcXdqXZxtqBgn5VXqx8bLhZh8N\":220,\"F7Wa1su7NRSr6LWuhPWdXcQALDyzm8Vmev7WtV5jVPtJELs\":320,\"F7ia6uitDknF5HhGQnTPuBdqFM3nTqpMP8oeWgjU4gWiaVm\":340,\"F8DXBsUQrvVvL9AvkEqxykmy1AWcBcn2CqjG3nyzYRFVJeS\":300,\"F8PTaGuZQo5fgRBFuhNnhd5euFiR3KLQNMVhYD5BduPKpHr\":300,\"FAGzHVggwv1QRmkGjom1Foc24jzZS1CJGcWUzrGdW8FyXEm\":380,\"FAYyBS6arn3X4fvtdybaBUdw5zqVsv3PPwRXXXYRTzTGFDv\":380,\"FAp5gVpwhwdmnJ1Ycet9z6Uva5tnfyCfA3AaY7WGi7u3DU4\":320,\"FAtJupteW5urNYVQvUcLaw2X9xtdwjumK2gvJpuBnAwFHvq\":320,\"FBichC4g5HBmdWCu3ebdADWQbvjbN7KdodNKWBYpLFgxCcd\":360,\"FChhpqk6Er57sXMC71F2aQ7EsNBQZTe3wUGXfKEvLH7W8zE\":340,\"FD3A8DXs16wGvcoi9stjpwz2pmxXFymTkVyVM4Tq5z6LWCK\":280,\"FDDy3cQa7JXiChYU2xq1B2WUUJBpZpZ51qn2tiN1DqDMEpS\":200,\"FFdDXFK1VKG5QgjvqwxdVjo8hGrBveaBFfHnWyz1MAmLL82\":260,\"FJ68vDYXj1A7QxHKf6j7qcCdShrU2o5A2sZ3y5w4Rst2d3a\":400,\"FJaSzBUAJ1Nwa1u5TbKAFZG5MBtcUouTixdP7hAkmce2SDS\":380,\"FKJNhxaXraoh85DRvChSzoDKHMAk9cZYxoq5LQW7uqeAQMD\":440,\"FLpAi7Zi4AB1JqJHWMrub4Umj4X73mZ5dAWE7Q6uS5e82pE\":320,\"FMTCvCG8qyWbRhb6k8Nu4nJkjYxnUpKY5QzSWfbt1fJH2K6\":380,\"FPRN9EfcCTswkRXfrkwoouep8uhVJRvMy1hnhB4Hb73biwK\":320,\"FRbyyQ55VCnCA3zXPjibseiAYKPhpVo5cDVFaboKw182qYb\":220,\"FSETB7JeTuTsJBYzUcKBtHXBYtBft3pZ87FUxP2GaY4acFh\":340,\"FUfXiFsaoWbUeaWiWBDizP7VbTSLxMirbJEpLAk1g6YueJp\":400,\"FX7rJbfiTFCqBuCWHLgR8SosDyZh842nv79mdQF4vxnZrhS\":300,\"FXCgfz7AzQA1fNaUqubSgXxGh77sjWVVkypgueWLmAcwv79\":460,\"FXezGfeLvze1DWUwwEc8MczpLSMnZrSDrcA7Ghyc84NaXGt\":320,\"Faaf2qgwtaFum78U8TdEPnUd4u7HPk65cdaFj7KbEA6wi2Y\":240,\"FcjmeNzPk3vgdENm1rHeiMCxFK96beUoi2kb59FmCoZtkGF\":280,\"Feu9bCUFaYSbkeqSY485kJjUoizp9CG4z3iSnGMwTqWSLeH\":280,\"Few2tfYD3wfyQXPCVkiqK8UGfuj5FK6ecadwfAmJQHBtASX\":300,\"Fgqjkry96qFLpRqPZstNzgaqKXiVyrpzTqD55neMdW8PK6g\":400,\"FhmaBgRFjZYFUWJp91ZHUbuW2Dm32VH4BVhnRQJqfpMmEok\":340,\"FiBHHbxDiVysvn5YBV78YFVUUykZRfQYhjxymu8VXnTqNMr\":220,\"FiHWU9AjN7z2no8zyiVEXtiTE46izxDgEcF1KM4r2r1GK59\":200,\"Fk3yTFztZdZa4a7yBpisz9ceMyjgYLtZ9CKSCfFNVhoW2ZC\":120,\"Fk6p456PTU6Sju2b83Cy8rU3NGFsXmWk9BrcqaMqhWW1jWf\":440,\"FmQHyUXoRkGTRySqVUy7NBAVhkKFvTtRtkzVTjZgBzbDzum\":260,\"Foc4anJqDfTMY3CvpCZzK9kUSudwc2oUTJ19K1E3JYFKy9j\":200,\"FpVmbKAqwoQaTBGwrTGsSu4nN1PCGY8XrQa4AaJVuzyVhh2\":300,\"FrL6qzw8bAFS6MTpbwLzEuAU9SCnALD5qMj9ZYFyt6zk5o4\":460,\"Fsspzse4QY1KqagdyrVqDt7cmVBr3HSVsfJ38WKgxsLVaXo\":340,\"FtqCc5yLcLc1FkLyftyNuCtYFCudo3unwwPQLzCbnWppjoc\":200,\"FwPTgWjafLgtxoRHK7pkUc2NJ9YzD7wHeCny2fMk41T9jKW\":340,\"Fy2rsYCoowQBtuFXqLE65ehAY9T6KWcGiNCQAyPDCkfpm4s\":200,\"G7Ur4BnMSfP2qE7ruSob5gwGQ5nzkGWu7Yqh14FcMqnDtgB\":260,\"G7eJUS1A7CdcRb2Y3zEDvfAJrM1QtacgG6mPD1RsPTJXxPQ\":180,\"G7mWyu1Pom5XreLHUzDEcvFp6WaMuLuo4QKxtDB9yJZnH69\":340,\"G866nkBvVKUqTFwAukeLM7n6QH9JyJDUJ2EvGFzKdymzqc3\":240,\"GAfhRsNqB9hwEmRFPhMZCvicFJ1kVtRF3UChYyKHq612ZV9\":260,\"GC8hwHbQ4TdbYJJPDS96G7Uj9bivnW5z56UEkqujjwhQPp5\":260,\"GCiTn1UJQT9TE5iamqoKweVoWGRZr9DF8uKXL4cG98fXczf\":300,\"GLiebiQp5f6G5vNcc7BgRE9T3hrZSYDwP6evERn3hEczdaM\":360,\"GRzsaGxLApkBfsUSCHKLnxhe4QECX4E2kQ5LuV4qqcijN2B\":200,\"GSWsxFWwvfpppd4q5iBa3fiBp3twEGvtYZFWNRVF17And24\":240,\"GTzRQPzkcuynHgkEHhsPBFpKdh4sAacVRsnd8vYfPpTMeEY\":160,\"GU7kw1ZYDKchyzhrAQ3imiY7s4fsoeFCvgeaU5XkTi9ABuB\":380,\"GXaUd6gyCaEoBVzXnkLVGneCF3idnLNtNZs5RHTugb9dCpY\":380,\"GZ5xCKCC8JRpJSMgAoDJAFAuQV8gD9Xz2cJca4mR7oRhzL9\":220,\"GbpLB9AZwVBBfecbMoTv3HCVb6irwN6y5YeA7uDNoUuRFrL\":380,\"Gc6YgfbTJ6pYXcwib6mk3KoiwncLm4dfdcN3nwFjvfi4Agd\":200,\"GcgPeEtLketwNDVVdV2jEnaTU5RMdGQdpYqVshssBWy1txZ\":200,\"GcqKn3HHodwcFc3Pg3Evcbc43m7qJNMiMv744e5WMSS7TGn\":320,\"GeYJhboY5bEc5WZFbrdxhEF9m6Y4NnbKzfCu1rBHxGWgviK\":220,\"GfXJERNZeUXh75pdFwksC9y42sftVti9K2AWntarJeM3gM4\":320,\"GffdZaTf5oBcHVfx4VjnA6AMpHEBa3PiaBzTBUPkt24fUxp\":340,\"GhoRyTGK583sJec8aSiyyJCsP2PQXJ2RK7iPGUjLtuX8XCn\":420,\"GiBnzCGFofhmAvsUv9FUShUb8YJYYwWex3ThQNkbDDNprS6\":260,\"Gk6v5CXUy2cPMtVxXtN7ZUn7K5y7UFEm78xp98Uatjt2yuV\":220,\"GkXpHnvjw7GQPTwMzy3d4MyZv8j2zFgiTtw1aqwfLhHdWqu\":180,\"GptHKfncW4Xwif6ohcZx6apt4B7jU7Pugvw1usQygP8WWQ7\":280,\"Gt6HqWBhdu4Sy1u8ASTbS1qf2Ac5gwdegwr8tWN8saMxPt5\":340,\"Gth5jQA6v9EFbpqSPgXcsvpGSrbTdWwmBADnqa36ptjs5m5\":300,\"H4V7fZJPLiGtBvQfsadb7oGfV1StiXJTuca76Daa449rz27\":140,\"H6bJZpE7PTFZgkLWGbeP3D1PqPGsFGqYAArFYa3ksCskmbt\":240,\"H9R6HgnZKtrcfBJP2M6WCvLJvp72Q96eURbCxmj6KCFVWjh\":240,\"H9bhxgaMbND1u7TXxmMpcQCTiLfQtGmdmwAyKkRnKsndSXa\":460,\"HA5jB52fFL1v4EoEHV4WgEiFZr7wGLiuBDZtSmnwKypvat7\":180,\"HAGcVQikZmEEgBBaChwjTVdwdA53Qopg2AYUtqw738C5kUq\":240,\"HHARzd4icVZULpMVEGZRiLhjfb3iDDohg1bdHRBusrMUvfd\":260,\"HJ1dGPxVr13KHGiCTGQfZjZMKGc8J52CsHEjcEXNMDyeGxf\":400,\"HP8qJ8P4u4W2QgsJ8jzVuSsjfFTT6orQomFD6eTRSGEbiTK\":260,\"HRMhY2CtVMp2yVSieKvq8Y8FHAhRKrGGMd6MoQe3iN6uJ2D\":340,\"HRuaGanNmkmeQgZPWPXmkZJb944raNS5ni2vhKzhz75zVYP\":280,\"HTrpbES27bqMvCioQGHpmJbBzwji6V5DeuXUfB1gsZ5Vkh1\":300,\"HWAGAxX2PAzNVg7w3ZyTprH5yvwbVwQ8rbWwuZxtQKbQupW\":340,\"HWpid7FWbuA7GxN1yykrYqSe6cY3ybUaX3qErCoiiFDSqSh\":240,\"Hadx1N8xZq6tRtXWkm6s5madXTXqVhauDNmesZnpMYTufcv\":200,\"HeKaZXya7rPQq7h7KgKnb3jG7n7VBFPr7PFTFYd6ZQbZRiU\":300,\"HeeJfizAEvorbkinL4GfRUYpxUiFST3dpnUHrh9ga2Z8Cpm\":300,\"HgWWnAXFGikrPVD2FrZ6CRk7KnYdVDn7zVyye8hqFPMc5g1\":320,\"HnnFkvtzrSrNpSUdG47E9ioBmDoDBnRUL6Lmy7GeqGcFc5c\":420,\"HqFz5RBczgtKrHQGav7DFZwSwrDXxKjLWc95mMDidVdfpwC\":300,\"HqGhgHg6YvnhaXSnaAUvyTDiR4FirB6Ssh2XNDedTzwCDv2\":180,\"Hqa9LGT3qF96agPYYbdfmUzh5P94MX9sqfF1WN4JnfRRVir\":360,\"HshTdrZiSJntTRh5oNytD2QuT38VDJHoGQmfcrtrZbViSGL\":200,\"HvjQwyG8NDFSD3ZP7VTbfBXHw4mZZd7LRMTcaj9TdfLynzu\":220,\"HxzXcFAM74evqTV3ZMDfM1Ec6zwez66MeBjc6XtdLTmsSFh\":380,\"HyTJYoYwA8GGgDH2XVnUhsPgMq7nnnE4Qq2BdwmpdHNzSdo\":160,\"J1aHBD7YQ3xnSwgFJNokjsEo7NjLTf36ZEHD2LTpeo7KAq8\":220,\"J2HVhQBYpx5PkyxHYLsp555pvWzc2zvGfNUnTwgzvRqVGqm\":220,\"J6RoE9MSpnZhX7hniJAJjBCcTqpr49SNhLvaVeJgFd69TJZ\":440,\"J6xn7Mr8pfed6gvvRPZ8HEEb89RCwheTBtxymg9Xw36hUUS\":460,\"JDA8ByXeJcn2BfNabC6WbBJKEBwM4k7oBVenVgJdzL32RJc\":200,\"JHTXKwYBhRZKWPW3nbtwtfE8dfRDYCS65cJXueMhg11k4cs\":320,\"JKjFSGnsXeqhhPgvBmKb6vUnrTZHu17eydEDnpcorCnZXLG\":240,\"JLKcPk652UTtQfyrk7keyU6vqAEj5JXzDAvQdPxB1DJTnZ3\":200}}",
        "block_number": 1459838,
        "session_index": 2585,
        "stash_id": "\"ESaTdKYYtRYhJbYjMzgKauNyZHbaNzx2vknC9eioqKusWSt\"",
        "timestamp": 1584269638
      },
```

**Comment by Bitcoinera** [comment]
> Hey @Bitcoinera!
> 
> I tested your branch and great! ^^ We are getting some rows in the rewards table!

Amazing! I will look into solving these issues one by one. But so happy it is finally saving data on Hasura 🎉

**Comment by mariopino** [comment]
This is working, can be cleaned up a bit but let test it before!

**Comment by mariopino** [comment]
Fixed some issues, data looks good, same as in Polkadot JS UI -> staking ^^

**Pull request merged**